File: api/Models/TradingSession.cs
namespace TradingDashboardApi.Models;

public class TradingSession
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string StartTime { get; set; } = string.Empty;
    public string EndTime { get; set; } = string.Empty;
    public TradingSessionStatus Status { get; set; }
    public List<string> Products { get; set; } = new();
    public string CreatedAt { get; set; } = string.Empty;
    public List<Offer> Offers { get; set; } = new();
}

File: src/app/models/trading-session.ts
export enum TradingSessionStatus {
  Active = 'active',
  Scheduled = 'scheduled',
  Completed = 'completed',
}

export interface TradingSession {
  id: number;
  name: string;
  startTime: string;
  endTime: string;
  status: TradingSessionStatus;
  products: string[];
  createdAt: string;
}
